chore: update docs for algo and hbar recoveries (#408)
BitGo/wallet-recovery-wizard: NON_BITGO_RECOVERY.md

17. For Avalanche C-Chain, the key is from https://snowtrace.io/.

> 18. For Algorand, clients are required to use their own node to build/broadcast recovery transactions.

Reviewer: ```suggestion
18. For Algorand, you must use your own node to broadcast recovery transactions.
```

> 19. The SDK determines the amount of Coin that can be recovered, and builds a fully signed transaction to send that amount to the Destination Address.
> 
> 20. This completed transaction can be broadcasted on the network using a blockchain explorer to finish the recovery.

Reviewer: ```suggestion
19. The SDK determines the amount of an asset that you can recover, and then builds a fully-signed transaction to a destination address.

20. To finish the recovery, broadcast the fully-signed transaction on the network using a blockchain explorer.
```
